calendar-bundle/contao/dca/tl_calendar_events.php

```diff
@@ -455,6 +455,15 @@
 			'eval'                    => array('tl_class'=>'w50'),
 			'sql'                     => "varchar(255) NOT NULL default ''"
 		),
+		'searchIndexer' => array
+		(
+			'filter'                  => true,
+			'inputType'               => 'select',
+			'options'                 => array('use_reader_page_setting', 'always_index', 'never_index'),
+			'eval'                    => array('maxlength'=>32, 'tl_class'=>'w50'),
+			'reference'               => &$GLOBALS['TL_LANG']['tl_calendar_events'],
```

Comment:
imho the reference should probably point somewhere to `MSC` in the `default` translation domain, to make things easier.

Reply:
Why do you want 4 options for the select? See https://github.com/contao/contao/pull/8252#issuecomment-2782410388

Reply:
May be, but then you only need specific translations for the `blankOptionLabel`. Or we don't use a `blankOptionLabel`, just like for the `robots` (`-` == fallback to page).

Reply:
> I think the blank label is important to avoid confusion for users, to make it clear which value is being used/applied

But why do we need it here and not for `robots` as well?

Reply:
> OK, I'll remove it and change it to be consisten with robots (no blank label). But don't blame me if people are getting confused 😉 Maybe it's enough to explained it in the documentation later.

Things can always be improved later. Keep it simple for now.

> Did you mean using no label for the blank option at News/Events/FAQ only, or remove it at Pages as well?

I think a `blankOptionLabel` on the page level wouldn't hurt. But we can always change these details later on and they are not important to get this draft review ready. Keeping it simple and consistent is more important imho.